### PR TITLE
Update recommended Doxygen code block from /// to /** */

### LINKED ISSUE #29: suggestion: endorsing or allowing alternate doc comment style for C++
The current recommendations state that `///` is recommended / preferred. I would like to propose the alternate style

```c++
/**
 *
 */
```

to be used or left as an option to the project creator/owner. This is probably mostly a matter of taste, but I find this style a little cleaner (whatever that means...).

### PR DESCRIPTION
**Pull request by tristan0x**
Fixes #29

### CODE REVIEW

**Comment by ferdonline** [comment]
I don't think we need to change the default. Im ok with `///`

**Comment by tristan0x** [comment]
> I don't think we need to change the default. Im ok with `///`

I am also ok with `///`. But the documentation should at least suggest what most developers prefer.

**Comment by ferdonline** [comment]
/** */ in single line comments is really ugly, I would not recommend that.
In the end it's a personal taste, and since `///` has some advantages I would suggest that as the default, but allowing /** as alternate style.

**Comment by matz-e** [comment]
> `/** */` in single line comments is really ugly, I would not recommend that.
In the end it's a personal taste, and since `///` has some advantages I would suggest that as the default, but allowing `/**` as alternate style.

Don't you have to use `///` in at least two subsequent lines? That's worse IMO.

**Comment by tristan0x** [comment]
> `/** */` in single line comments is really ugly, I would not recommend that.
> In the end it's a personal taste, and since `///` has some advantages I would suggest that as the default, but allowing `/**` as alternate style.

I agree with you but like you said it is a matter of personal taste and this guideline should reflect what most people do.

File: cpp/Documentation.md [hunk 1]
```diff
@@ -57,17 +61,19 @@ using the `\internal` Doxygen command.
 
 
 ```c++
-/// Helper function to create a vertex unique identifier.
-/// \param type vertex type
-/// \param id vertex identifier
-/// \return vertex unique identifier
+/**
+ *  Helper function to create a vertex unique identifier.
```
Comment (ohm314) [inline cpp/Documentation.md]:
```suggestion
 * Helper function to create a vertex unique identifier.
```

**Pull request merged**